app/src/androidTest/java/com/blackboxteam/app/blackbox/LaunchScreenTest.java

```diff
@@ -32,11 +32,20 @@
             LaunchScreenActivity.class);
 
     @Test
-    public void navigateToLogin () {
+    public void navigateToLoginScreen () {
         // Clicks a button to navigate to login screen
         Intents.init();
         onView(withId(R.id.launchScreenLoginButton)).perform(click());
         intended(hasComponent(LoginScreenActivity.class.getName()));
         Intents.release();
     }
+
+    @Test
+    public void navigateToSignupScreen () {
+        // Clicks a button to navigate to signup screen
+        Intents.init();
```

Comment:
Consider moving this to the setup method annotated with `@Before`. This method runs before each test, so we can move initializations there. e.g
At the top of all `@Test` methods, write this:
`@Before ``
public void setup() throws Exception {
    //initialize
}`